.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 go:
-- 1.6.3
 - 1.7
+- 1.8
```

Comment:
Was actually my plan. Thank you for the quick merge. What do you think about this pull request? Is there anything I need to do?